test/curator_test.rb

```diff
@@ -0,0 +1,191 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/photograph'
+require './lib/artist'
+require './lib/curator'
+
+class CuratorTest < Minitest::Test
+
+  def setup
+    @curator = Curator.new
+    @photo_1 = Photograph.new({id: "1", name: "Rue Mouffetard, Paris (Boy with Bottles)", artist_id: "1", year: "1954"})
+    @photo_2 = Photograph.new({id: "2", name: "Moonrise, Hernandez", artist_id: "2", year: "1941"})
+    @photo_3 = Photograph.new({id: "3", name: "Identical Twins, Roselle, New Jersey", artist_id: "3", year: "1967"})
+    @photo_4 = Photograph.new({id: "4", name: "Monolith, The Face of Half Dome", artist_id: "3", year: "1927"})
+    @artist_1 = Artist.new({id: "1", name: "Henri Cartier-Bresson", born: "1908", died: "2004", country: "France"})
+    @artist_2 = Artist.new({id: "2", name: "Ansel Adams", born: "1902", died: "1984", country: "United States"})
+    @artist_3 = Artist.new({id: "3", name: "Diane Arbus", born: "1923", died: "1971", country: "United States"})
+  end
+
+  def test_it_exists
+    assert_instance_of Curator, @curator
+  end
+
+  def test_it_has_photographs
+    assert_equal [], @curator.photographs
+  end
+
+  def test_it_has_photographs
+    @curator.add_photograph(@photo_1)
+    @curator.add_photograph(@photo_2)
+
+    assert_equal [@photo_1, @photo_2], @curator.photographs
+  end
+
+  def test_it_has_artists
+    assert_equal [], @curator.artists
+  end
+
+  def test_it_can_add_artists
+    @curator.add_artist(@artist_1)
+    @curator.add_artist(@artist_2)
+
+    assert_equal [@artist_1, @artist_2], @curator.artists
+  end
+
+  def test_it_can_find_artist_by_id
```

Comment:
This test is passing only because you have given it the incorrect expected value, and the curator only knows about this one artist.  A better test (one that would help you debug where your method is broken), would look like this:

```ruby
@curator.add_artist(@artist_1)
@curator.add_artist(@artist_2)
# Now the curator is looking for a single artist from a list, rather than looking for the only artist that the curator knows about.

assert_equal @artist_1, @curator.find_artist_by_id("1")
# Based on the interaction pattern, we are looking for a single artist object, not an array of objects.
```

It looks like the rest of your tests have these kinds of errors.  They are passing, but shouldn't be.
```